clang/test/CIR/CodeGen/aarch64-neon-intrinsics.c

```diff
@@ -2839,79 +2839,103 @@
 //   return vrhaddq_u32(v1, v2);
 // }
 
-// NYI-LABEL: @test_vqadd_s8(
-// NYI:   [[VQADD_V_I:%.*]] = call <8 x i8> @llvm.aarch64.neon.sqadd.v8i8(<8 x i8> %a, <8 x i8> %b)
-// NYI:   ret <8 x i8> [[VQADD_V_I]]
-// int8x8_t test_vqadd_s8(int8x8_t a, int8x8_t b) {
-//   return vqadd_s8(a, b);
-// }
 
-// NYI-LABEL: @test_vqadd_s16(
-// NYI:   [[TMP0:%.*]] = bitcast <4 x i16> %a to <8 x i8>
-// NYI:   [[TMP1:%.*]] = bitcast <4 x i16> %b to <8 x i8>
-// NYI:   [[VQADD_V2_I:%.*]] = call <4 x i16> @llvm.aarch64.neon.sqadd.v4i16(<4 x i16> %a, <4 x i16> %b)
-// NYI:   [[VQADD_V3_I:%.*]] = bitcast <4 x i16> [[VQADD_V2_I]] to <8 x i8>
-// NYI:   ret <4 x i16> [[VQADD_V2_I]]
-// int16x4_t test_vqadd_s16(int16x4_t a, int16x4_t b) {
-//   return vqadd_s16(a, b);
-// }
```

Comment:
Thanks for cleaning up this file, one remaining change: this should also go to the subdir AArch64, perhaps call it `neon.c`.